alarm/PolicyManager2.js

```diff
@@ -115,33 +113,35 @@ class PolicyManager2 {
     })
 
     this.queue.process((job, done) => {
-      const event = job.data
-      const policy = this.jsonToPolicy(event.policy)
-      const oldPolicy = this.jsonToPolicy(event.oldPolicy)
+      const event = job.data;
+      const policy = event.policy instanceof Policy ? event.policy : new Policy(event.policy);
```

Comment:
event.policy will never be a class, because it's passed from redis pub/sub

Reply:
not all, after this change...
https://github.com/firewalla/firewalla/blob/b5b7fdf26d4ca67e5d1163937e7dca4587e1a1a1/alarm/PolicyManager2.js#L712-L726